app/Environment.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App;
+
+trait Environment
```

Comment:
i prefer verb phrases (?) for my trait names. `DoesThings`.

Could this be `DetectsEnvironment` or something?

Reply:
Done.